lib/crypto/blsContainter.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable */
```

Comment:
I think we can remove this line from this file, it uses some new features anyway, like arrow functions for example